public/app/features/search/page/components/ManageActions.tsx

```diff
@@ -43,30 +44,17 @@ export function ManageActions({ items, folder, onChange }: Props) {
     setIsDeleteModalOpen(true);
   };
 
-  const onToggleAll = () => {
-    alert('TODO, toggle all....');
-  };
-
   return (
     <div className={styles.actionRow}>
       <div className={styles.rowContainer}>
         <HorizontalGroup spacing="md" width="auto">
-          <Checkbox value={false} onClick={onToggleAll} />
+          <IconButton name={'check-square' as IconName} onClick={clearSelection} title="Uncheck everything" />
```

Comment:
🤔 I think we might need the `onToggleAll` after all,  because the folder view page (it has a checkbox to "select all")
![oldGoToFolder](https://user-images.githubusercontent.com/239999/169852714-60f1c772-c004-4ce4-8367-cebf90e210ba.png)


In the new folder page, we don't offer that option

![newGoToFolder](https://user-images.githubusercontent.com/239999/169853625-cde0bdb3-4469-4926-99f6-da2657c03fc2.png)

Reply:
Maybe... I get pretty nervous about our "select all" button for the actions we are taking :) works OK when the numbers are small, but pretty sketchy when the numbers are large 😬   When small, checking everythign is not hard too!

Given the beta tomorrow... I think we should mege this and add a TODO item/discussion if we want a select all.